src/modules/components/proposal/proposalActions/proposalActions/proposalActions.tsx

```diff
@@ -54,30 +58,46 @@ export const ProposalActions: React.FC<IProposalActionsProps> = (props) => {
 
     return (
         <Card className={classNames('w-full overflow-hidden', className)}>
-            <Heading size="h2" className="p-4 md:p-6" ref={actionsContainerRef}>
-                {copy.proposalActionsContainer.containerName}
-            </Heading>
-            <Accordion.Container isMulti={true} value={expandedItems} onValueChange={handleAccordionValueChange}>
-                {actions.map((action, index) => (
-                    <ProposalActionsAction
-                        key={`action-${index}`}
-                        action={action}
-                        index={index}
-                        name={actionNames?.[action.type]}
-                        CustomComponent={customActionComponents?.[action.type]}
-                        {...web3Props}
+            {heading && (
+                <Heading size="h2" className="border-b border-neutral-100 p-4 md:p-6">
+                    {heading}
+                </Heading>
+            )}
```

Comment:
Should this be the section heading we display on the app (Actions + description)? In that case we can just use the Page.Section from the app and remove this custom `heading` component here